lib/index.test.js

```diff
@@ -129,7 +130,46 @@ test('should render invisible recaptcha', t => {
       callback: onVerify,
       'expired-callback': onExpire,
       'error-callback': onError,
-      isolated: true
+      isolated: true,
+      hl: null
+    })
+  );
+});
+
+test('should render recaptcha with hl', t => {
+  t.plan(2);
+
+  const onVerify = sinon.stub();
+  const onExpire = sinon.stub();
+  const onError = sinon.stub();
+
+  mount(
+    <Reaptcha
+      sitekey="my-key"
+      theme="dark"
+      size="normal"
+      hl='fr'
+      tabindex={2}
+      onVerify={onVerify}
+      onExpire={onExpire}
+      onError={onError}
+      explicit={false}
+    />
+  );
+
+  t.true(renderSpy.calledOnce);
+  t.true(
+    renderSpy.calledWith(sinon.match.any, {
```

Comment:
This test checks if recaptcha renders with the `hl` property. All other properties are unnecessary. Try this:
```
t.plan(1);
(...)
t.true(renderSpy.calledWith(sinon.match.any, sinon.match({ hl: 'fr' }));